WORKSPACE

```diff
@@ -85,6 +85,14 @@ go_repository(
     version = "v0.0.0-20200218151345-dad8c97a84f5",
 )
 
+go_repository(
```

Comment:
Seeing as how this is already specified in go.mod, it's also already present in DEPS.bzl.
```
    go_repository(
        name = "com_github_mmatczuk_go_generics",
        build_file_proto_mode = "disable_global",
        importpath = "github.com/mmatczuk/go_generics",
        sum = "h1:QjLgi/L+MjxysinrA8KkNZLf3cAhTluBoSXUvFeN144=",
        version = "v0.0.0-20181212143635-0aaa050f9bab",
    )
```